src/community/call-for-speakers-2024/index.md

```diff
@@ -1,126 +1,7 @@
----
-title: 2024 Call for speakers
-description: Description
-section: Community
-theme: Events and workshops
-layout: layout-pane.njk
-order: 16
----
-
-{% from "_call-to-action.njk" import callToAction %}
-{% from "_promo-banner.njk" import promoBanner %}
-
-<p class="govuk-!-font-size-24">
-  This year our in-person Design System Day event will be held on <strong>5 September 2024</strong>. We’re excited to announce that we will be holding the event at <a href="https://www.stgeorgeshallliverpool.co.uk/" class="govuk-link">St George’s Hall</a> in Liverpool.
-</p>
-
-<img class="app-image--no-border govuk-!-margin-top-9 govuk-!-margin-bottom-9" src="/images/dsd24-hello.svg" alt="Hello">
-
-As ever, we want the conference to be a community-driven event. So come and take part to make Design System Day 2024 special.
-
-We're looking for keynote speakers, as well as folks to host show and tell sessions, panel discussions and talks throughout the day.
-
-{% call promoBanner({
-  img: "/images/dsd24-star.svg",
-  padding: 2,
-  marginBottom: 7
-}) %}
-
-  <p>
-    We appreciate diverse perspectives and would love to hear from first-time speakers.
-  </p>
-{% endcall %}
-
-## Details
-
-<table class="govuk-table">
-  <tbody>
-  <tr class="govuk-table__row">
-    <th scope="row" class="govuk-table__header">
-      Date
-    </th>
-    <td class="govuk-table__cell">
-      Thursday 5 September 2024
-    </td>
-  </tr>
-  <tr class="govuk-table__row">
-    <th scope="row" class="govuk-table__header">
-      Location
-    </th>
-    <td class="govuk-table__cell">
-      St Georges Hall, Liverpool
-    </td>
-  </tr>
-  <tr class="govuk-table__row">
-    <th scope="row" class="govuk-table__header">
-      Time
-    </th>
-    <td class="govuk-table__cell">
-      9:45am to 4:30pm
-    </td>
-  </tr>
-  </tbody>
-</table>
-
-If you’re interested in running either a show and tell, panel discussion or talk, now is your opportunity. Send us the details of your pitch by 11:59pm on Sunday 28 July 2024. Any pitches submitted after this date will not be assessed as we will not have the capacity to do so.
-
-### Theme
-
-Our theme this year is ‘Unsung heroes of the Design System’. We’d especially like to hear from folks working in a content or product role, as these are often less heard from.
-
-### Length
-
-Sessions are 45 minutes long. If you need longer let us know.
-
-### Title
-
-- make it clear in the title what the workshop is about
-- make your title engaging
+title: This page has been archived
```

Comment:
@Ciandelle  The page properties are missing a leading `---`, which makes the [list of properties show directly on the page](https://deploy-preview-4043--govuk-design-system-preview.netlify.app/community/call-for-speakers-2024/) rather than being used by the site to set the title and remove from the sitemap.

This might be the cause of [the tests currently failing](https://github.com/alphagov/govuk-design-system/actions/runs/10420335081/job/28860185420?pr=4043), so it's worth trying to add that back first before trying to figure where the empty link is 😊

Reply:
```suggestion
---
title: This page has been archived
```

Reply:
And it seems it was what was making the tests fail, as the page was still included in the sitemap but had no title because no properties was attached to it.

Reply:
Ah awesome, thank you @romaricpascal it looks like everything has now passed, are you happy to approve?